docs/self-managed/operational-guides/application-configs.md

```diff
@@ -0,0 +1,483 @@
+      zeebeElasticsearch:
+        # Cluster name
+        clusterName: elasticsearch
+        # Host
+        host: cpt-elasticsearch
+        # Transport port
+        port: 9200
+        # Index prefix, configured in Zeebe Elasticsearch exporter
+        prefix: zeebe-record
+    logging:
+      level:
+        ROOT: INFO
+        io.camunda.operate: INFO
+    #Spring Boot Actuator endpoints to be exposed
+    management.endpoints.web.exposure.include: health,info,conditions,configprops,prometheus,loggers,usage-metrics,backups
+```
+
+Then, take the contents under `application.yml` and put it under the `operate.configuration` section in `values.yaml`.
+
+## Where to search for configuration options
+
+- [Zeebe Broker](docs/self-managed/zeebe-deployment/configuration/broker.md)
+- [Zeebe Gateway](docs/self-managed/zeebe-deployment/configuration/gateway.md)
+- [Operate](docs/self-managed/operate-deployment/operate-configuration.md)
+- [Tasklist](docs/self-managed/tasklist-deployment/tasklist-configuration.md)
+- [Web Modeler](docs/self-managed/modeler/web-modeler/configuration/configuration.md)
+- [Console](docs/self-managed/console-deployment/configuration.md)
+- [Connectors](docs/self-managed/connectors-deployment/connectors-configuration.md)
+- [Identity](docs/self-managed/identity/deployment/configuration-variables.md)
+- [Optimize]($optimize$/self-managed/optimize-deployment/configuration/system-configuration)
+
+## Are there any drawbacks to specifying a config file in this way?
+
+Customizing the `configuration` option will replace the entire contents of the configuration file. During upgrades, if the `configuration` option remains and if there are any application-level breaking changes to the configuration file format, this may cause the application component to crash.
+
+## What will happen if I set an environment variable AND a configuration file?
+
+Suppose we have a `values.yaml` with the following:
+
+```yaml
+zeebe:
+  env:
+    - name: ZEEBE_BROKER_DATA_BACKUP_S3_BUCKETNAME
+      value: "zeebetest1"
+
+  configuration: |
+    zeebe:
+      broker:
+        data:
+          backup:
+            s3:
+              bucketName: "zeebeOtherBucketName"
+      ...
+```
+
+Notice how both the environment variable and the configuration file are configuring the same option with conflicting settings. The environment variable has the bucket name set as `zeebetest1` and the `configuration` option has the bucket name as `zeebeOtherBucketName`. Which option will override the other?
+
+In this case, the environment variable will take priority, because in the [Spring Documentation: Externalized Configuration](https://docs.spring.io/spring-boot/docs/1.5.6.RELEASE/reference/html/boot-features-external-config.html):
+
+> 10. OS environment variables
+
+is higher in the list than
+
+> 12. Profile-specific application properties outside of your packaged jar (application-{profile}.properties and YAML variants)
```

Comment:
This is a bit hard to read, let's move it between backticks "\`...\`" in one line.